guides/troubleshooting/README.md

```diff
@@ -0,0 +1,72 @@
+# Troubleshooting
+
+While deploying your app or making changes, you may run into a situation where your app will not deploy. This can happen for a number of reasons, and in this guide, you'll learn a few things that may help you get moving again.
+
+## Check the build logs
+
+The platform will pull your app from GitHub and build it when you first create it and anytime you make changes and push them to GitHub. During the build, the platform will run `npm i` and `npm run wpe-build`. It's possible that either for these could fail, and your build logs will tell you what happened.
+
+First, list the builds for an Environment of an App:
```

Comment:
```suggestion
To retrieve the build log for a failed build, you'll first need the Environment ID: 
```

I would suggest starting with `apps get` and follow up with, "If you don't recall the name of your App, you can see all of your Apps by `wpe alpha apps list`.

Reply:
Update